src/main/kotlin/no/nav/familie/ba/soknad/api/helsesjekk/HelsesjekkController.kt

```diff
@@ -21,6 +22,15 @@ class HelsesjekkController {
 
     @GetMapping("mottak")
     fun pingMottak(): Ressurs<String> {
-        return Ressurs.success("OK")
+        return Result.runCatching { mottakClient.ping() }
```

Comment:
Skulle vi lagt til en exception handler slik vi har gjort i ba-sak istedet for å catche alt?